src/Html.php

```diff
@@ -3944,7 +3944,8 @@ public static function initEditorSystem($id, $rand = '', $display = true, $reado
 
                      editor.on('submit', function (e) {
                         if ($('#$id').val() == '') {
-                           alert(__('The description field is mandatory'));
+                           const field = $('#$id').parent().parent().find('label').text().replace('*', '').trim();
+                           alert(__('The "' + field + '" field is mandatory'));
```

Comment:
Better to use `sprintf()` with a `%1$s` value for replacement. Duno exactly what is the best option for JS, but maybe something like `sprintf('The %1$s field is mandatory', 'PLACEBO')` and then use replace JS method to set correct value for `PLACEBO`.

Reply:
`__('The %s field is mandatory').replace('%s', field)`

Reply:
Good catch, it would mess up the translations otherwise.
Done 43a8f95794.